zebrad/src/commands/start.rs

```diff
@@ -90,6 +90,9 @@ impl StartCmd {
             ChainSync::new(&config, peer_set.clone(), state.clone(), chain_verifier);
 
         info!("initializing mempool");
+
+        let (transaction_sender, transaction_receiver) = tokio::sync::watch::channel(None);
```

Comment:
Readability: can you rename these channels so it's clear they're for the mempool?

Reply:
I am not sure if you mean change the name of this variables just here or everywhere. Also not sure if you want `mempool_transaction_sender` as the name or what exactly. In my opinion this names are pretty clear as they can be found always in context but open to change them if i know specifically to what.

Reply:
I think it's ok to just change them in this file - all the other uses are in tests, or in the `mempool` module.

This is the only place where a future refactor might confuse mempool and state channels, and cause a bug.

```suggestion
        let (mempool_transaction_sender, mempool_transaction_receiver) = tokio::sync::watch::channel(None);
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/2823/commits/ac7e32ed86be61162fa68e2051b7af790bef022e